src/Sylius/Bundle/VariationBundle/DependencyInjection/Configuration.php

```diff
@@ -202,6 +205,43 @@ private function addClassesSection(ArrayNodeDefinition $node)
                                             ->end()
                                         ->end()
                                     ->end()
+                                    ->arrayNode('translation')
+                                        ->isRequired()
+                                        ->addDefaultsIfNotSet()
+                                        ->children()
+                                            ->variableNode('option_value')->end()
+                                            ->arrayNode('classes')
+                                                ->isRequired()
+                                                ->addDefaultsIfNotSet()
+                                                ->children()
+                                                    ->scalarNode('model')->defaultValue(OptionValueTranslation::class)->cannotBeEmpty()->end()
+                                                    ->scalarNode('interface')->defaultValue(OptionValueTranslationInterface::class)->cannotBeEmpty()->end()
+                                                    ->scalarNode('controller')->defaultValue(ResourceController::class)->cannotBeEmpty()->end()
+                                                    ->scalarNode('repository')->cannotBeEmpty()->end()
+                                                    ->scalarNode('factory')->defaultValue(Factory::class)->end()
```

Comment:
Configuration of [option node](https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/VariationBundle/DependencyInjection/Configuration.php#L116) is wrong. With refactoring we forgot to add this statement and we didn't catch it during review. It is not really bad mistake, but we try to fix it anyway. Thats why every new resources should have this option. 
Another thing is TranslatableFactory. You are correct, that option node has different factory, but there you have **translatable resource**. Here, you have to specify factory for **translation**. Do you understand the difference? If you check few lines below configuration you've referenced, you will see [translation configuration](https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/VariationBundle/DependencyInjection/Configuration.php#L149) which contains regular factory class.